webapi/src/it/scala/org/knora/webapi/ITSpec.scala

```diff
@@ -60,11 +62,27 @@ class ITSpec(_system: ActorSystem) extends Core with KnoraService with Suite wit
     implicit protected val ec: ExecutionContextExecutor = system.dispatcher
     implicit protected val materializer = ActorMaterializer()
 
-    def singleAwaitingRequest(request: HttpRequest, duration: Duration = 3.seconds): HttpResponse = {
+    protected def singleAwaitingRequest(request: HttpRequest, duration: Duration = 5.seconds): HttpResponse = {
```

Comment:
I increased it to 15 seconds and they worked fine. 

I ran the tests on my MacBookAir.